torchvision/prototype/transforms/_misc.py

```diff
@@ -62,13 +62,14 @@ def __init__(self, transformation_matrix: torch.Tensor, mean_vector: torch.Tenso
         self.transformation_matrix = transformation_matrix
         self.mean_vector = mean_vector
 
-    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
+    def forward(self, *inputs: Any) -> Any:
+        sample = inputs if len(inputs) > 1 else inputs[0]
+        if has_any(sample, PIL.Image.Image):
+            raise TypeError("LinearTransformation does not work on PIL Images")
 
-        if isinstance(inpt, features._Feature) and not isinstance(inpt, features.Image):
-            return inpt
-        elif isinstance(inpt, PIL.Image.Image):
-            raise TypeError("Unsupported input type")
+        return super().forward(sample)
 
+    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
```

Comment:
Nit:
```suggestion
    def _transform(self, inpt: Any, params: Dict[str, Any]) -> torch.Tensor:
```

Reply:
I don't think `mypy` will complain here since we are making the type more concrete, but it won't have any effect either. This is an internal method and the user code will only "see" 

https://github.com/pytorch/vision/blob/becaba0e0546f434be64de4e9603ab66e4b160b2/torchvision/prototype/transforms/_transform.py#L28

Reply:
@pmeier I agree. I think it's worth it for someone who reads the code to see what we are currently returning.